src/main/scala/StepStatus.scala

```diff
@@ -169,6 +171,10 @@ class StepStatus(val stepNumber: String, val stepId: String, props: Properties)
     }
   }
 
+  def stepHasReduceStage(hss: HadoopStepStats): Boolean = {
+    hss.getCounterValue("org.apache.hadoop.mapreduce.JobCounter", "TOTAL_LAUNCHED_REDUCES").toLong > 0L
```

Comment:
Is this the right counter?  It looks like this only increments when reduce tasks actually launch